src/main/resources/static/faq.js

```diff
@@ -0,0 +1,32 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Eine so kleine JS Datei kannst du direkt in den Script tag im HTML packen. Macht nicht viel aus, aber sollte etwas schneller laden.